value.go

```diff
@@ -53,6 +53,15 @@ func Null(iso *Isolate) *Value {
 	return iso.null
 }
 
+func NewStringFromByteArray(iso *Isolate, val []byte) (*Value, error) {
```

Comment:
This could use a documentation comment.  A CHANGELOG.md entry would also be appreciated to mention the addition of this new method